services/matrix/matrix.service.js

```diff
@@ -128,17 +180,17 @@ module.exports = class Matrix extends BaseJsonService {
   static get route() {
     return {
       base: 'matrix',
-      format: '([^/]+)/([^/]+)',
-      capture: ['roomId', 'host'],
+      format: '([^/]+)(?:/([^/]+))?',
+      capture: ['roomAlias', 'serverFQDN'],
     }
   }
 
   static get examples() {
     return [
       {
         title: 'Matrix',
-        exampleUrl: '!ltIjvaLydYAWZyihee/matrix.org',
-        pattern: ':roomId/:host',
+        exampleUrl: 'twim:matrix.org',
```

Comment:
Can we use `namedParams` here instead of `exampleUrl`? See https://github.com/badges/shields/pull/2308 for more info 
```js
namedParams: {
  roomAlias: 'twin',
  'server_fqdn_optional': 'matrix.org',
}
```

Reply:
Fixed in https://github.com/badges/shields/pull/2527/commits/3ac4a25c726451687a3bb197d021f156a3012339 (and https://github.com/badges/shields/pull/2527/commits/d655fa146346ce882ddb704b89b3dd719a074c85)